### PR TITLE
Document the new Intl exists() methods

### PR DESCRIPTION
**Pull request by javiereguiluz**
As mentioned by our friend @ro0NL in this comment: https://symfony.com/blog/new-in-symfony-4-3-simpler-access-to-intl-data#comment-23227

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Comment by noniagriconomie** [comment]
I think it will be very usefull to have it the blogpost also :)

File: components/intl.rst [hunk 1]
```diff
@@ -240,6 +240,10 @@ which defaults to the current default locale::
     $languages = Languages::getNames('de');
     // => ['ab' => 'Abchasisch', ...]
 
+You can also check if a given language code is valid::
+
+    $isValidLanguage = Languages::exists($languageCode);
```
Comment (OskarStark) [inline components/intl.rst]: $languageCode is undefined, maybe use a real string here?

Comment (javiereguiluz) [reply components/intl.rst]: If I use a real string ... then I need to add a comment that the string argument is the "language code". Also, if we add a real string, we may need to show two cases: valid and invalid strings.

Do you really think that someone reading that code can't understand that `$languageCode` is a variable where you store the language code to check if it's valid?

Comment (OskarStark) [reply components/intl.rst]: Fair enough, your explanation makes sense to me, so let’s keep it like this 🚀

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@noniagriconomie the blog post has been updated: https://symfony.com/blog/new-in-symfony-4-3-simpler-access-to-intl-data

**Pull request merged**